### ISSUE TITLE
Changing hydrogen central production cost for wind and solar technologies

### ISSUE THREAD

**Issue by ysimsek10** [JGCRI/gcam-core#338]
Hi,
I want to change hydrogen central production cost for wind and solar technologies. When I check “Hydrgogen.xml” file I see 2 costs as follows: 
<minicam-non-energy-input name="electrolyzer"> 
                                <input-cost>2.2409</input-cost> 
                            </minicam-non-energy-input> 
                            <minicam-non-energy-input name="wind turbines"> 
                                <input-cost>8.1964</input-cost> 
                            </minicam-non-energy-input>  
For electrolyzer cost (non-energy cost), I think I can change from ‘H2A_electrolyzer_NEcost_CF.csv’ files however the second part of the cost, which is wind turbines in this example, I could not find which csv file/s are related to this cost. Also, I assume this is not non-energy cost although it is written non-energy input?

Thanks for your help!
Best.

**Comment by Youyi77** [comment]
I think that one is in _A25.globaltech_cost_?

**Comment by pkyle** [comment]
The "wind turbines" portion of the capital cost is inherited from the electric power sector where there are explicit assumptions about capital costs, fixed charge rates, fixed O&M costs, base capacity factors (i.e., the capacity factor of the least-cost turbines installed in the region), and supply curves, in each region and time period. Wind for hydrogen electrolysis and for the general electricity market share the same set of assumptions and resource bases in each region and time period. There isn't any built-in capability for users to assign the costs of wind or solar electrolysis independently, because these assumptions should always be consistent with the corresponding assumptions for producing electricity in general.

**Comment by Youyi77** [comment]
Hi @pkyle, a quick follow-up on this. So does it make sense to change the wind/solar electricity inputs (either efficiency or costs) to simulate an aggressive development of wind/solar electricity, thus stimulating the relevant H2 central production?

I noticed there are `solar_adv.xml` and `wind_adv.xml` which already fulfill the goal. (Seems that they are mainly used for SSP simulations)

Also wondering in the default `configuration` file, why the `electricity` line is commented but not activated?

![image](https://github.com/JGCRI/gcam-core/assets/156739336/558f842e-f03d-43ef-bdd1-450ecdf2de9d)


Thank you!

**Comment by Youyi77** [comment]
Just realized: `electricity_water` contains the info for electricity.

**Comment by pkyle** [comment]
Yeah that's right, the `electricity.xml` file contains a representation of the power sector that doesn't take water inputs or have cooling system types disaggregated. It hasn't been tested in years so I don't advise trying to run with that.
I think if you want to switch to using the "advanced" assumptions in `solar_adv.xml` and `wind_adv.xml` for the hydrogen production sector you'd need to switch out the data tables in the hydrogen code file `zenergy_L225.hydrogen.R` which only pulls in the reference assumptions in `L223.GlobalIntTechCapital_elec` with the ones used to build those advanced XML files. 
```
             "L223.GlobalIntTechCapital_sol_adv",
             "L223.GlobalIntTechCapital_wind_adv",
```

**Comment by Youyi77** [comment]
Thanks, @pkyle  Got it!! Appreciate the advice!
